src/main/java/io/github/genomicdatainfrastructure/daam/exceptions/AcceptTermsException.java

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package io.github.genomicdatainfrastructure.daam.exceptions;
+
+import java.util.List;
+
+public class AcceptTermsException extends RuntimeException {
+
+    private static final String MESSAGE = "Could not be accept terms for application %s due to the following errors:";
```

Comment:
important: please rephrase it to `Terms and Licenses of application %s could not be accepted, due to the following errors:`